test/utils.jl

```diff
@@ -0,0 +1,136 @@
+using ADNLPModels, NLPModels, NLPModelsJuMP, OptimizationProblems
+
+function set_meta(all::AbstractVector{T}) where T <: Union{Symbol, String}
+  for name in string.(all)
+    io = open("src/Meta/" * name * ".jl", "w")
+    write(io, generate_meta(name))
+    close(io)
+  end
+end
+
+"""
+  `generate_meta(model, name)`
+  `generate_meta(name)`
+  
+  is used to generate the meta of a given NLPModel.
+
+Default values for undetermined entries:
```

Comment:
We should probably add the description of these to the README (and docs?)

Reply:
Once the meta is added to this repo, it will be the end of a series of large changes. So, I planned to update the README and the docs at the end.

Reply:
I already have a draft for the doc [here](https://juliasmoothoptimizers.github.io/ADNLPModelProblems.jl/dev/tutorial/)

Reply:
:+1: